codecs/packbits/schema.json

```
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "name": {
+      "const": "packbits"
+    },
+    "configuration": {
+      "type": "object",
+      "properties": {
+        "padding_encoding": {
+          "enum": ["start_byte", "end_byte", "none"],
+          "default": "none"
+        }
+      },
+      "additionalProperties": false
+    }
+  },
+  "required": ["name"],
+  "additionalProperties": false
+}
```

I guess, if the configuration is optional, a string literal as name should also be permissable. I guess the same would be true for the dtypes as well.

```suggestion
{
  "$schema": "https://json-schema.org/draft/2020-12/schema",
  "anyOf": [{
    "type": "object",
    "properties": {
      "name": {
        "const": "packbits"
      },
      "configuration": {
        "type": "object",
        "properties": {
          "padding_encoding": {
            "enum": ["start_byte", "end_byte", "none"],
            "default": "none"
          }
        },
        "additionalProperties": false
      }
    },
    "required": ["name"],
    "additionalProperties": false
  }, {
    "const": "packbits"
  }]
}
```